### DISCUSSION TITLE
RGraph on cdnjs (again)

### DISCUSSION THREAD

**Discussion by heyesr** [cdnjs/cdnjs#14058] [General] [answered]
Hi,

I've previously asked about hosting RGraph libraries on cdnjs but at the time I didn't have a GitHub repository.

Now I do though and reading the policy on hosting libraries it says you may be flexible on the minimum 200 star requirement. As I write this my GitHub RGraph repository has a massive 14 stars (!) so my question is "Would you be flexible in my case?".

https://github.com/heyesr/rgraph

Thanks!

**Comment by MattIPv4** [comment, marked as answer]
Sure, feel free to open a PR on cdnjs/packages to add it :)

**Reply by heyesr** [reply]
Great - thanks! I'll do that (after reading more on how to use Git!).

**Reply by heyesr** [reply]
Hi,

OK I've forked the cdnjs/packages and edited the cdnjs/packages/r/RGraph.json file and you see that here:

https://github.com/heyesr/packages/blob/master/packages/r/RGraph.json

Is this correct? Should I go ahead and create a pull request ?

Thanks.

**Reply by heyesr** [reply]
PS. I have really no idea when it comes to working with Git - you might be able to tell!

**Reply by MattIPv4** [reply]
If you open the PR our automated CI should help with reviewing and making sure it's correct, and I can then also leave a review as well :)

**Comment by heyesr** [comment]
Hi

OK will do, cheers.

Richard
